pages/instructorAssessmentInstances/instructorAssessmentInstances.ejs

```diff
@@ -129,29 +212,63 @@
                 <% } %>
               <tr>
                 <% if (assessment.group_work) { %>
-                  <td><%= row.group_name %></td>
-                  <td class="text-center" style ="word-break:break-all;"><small><%= row.uid_list %></small></td>
+                  <td class="align-middle"><%= row.group_name %></td>
+                  <td class="text-center align-middle" style ="word-break:break-all;"><small><%= row.uid_list %></small></td>
                 <% } else { %>
-                <td><%= row.uid %></td>
-                <td><%= row.name %></td>
-                <td class="text-center"><%= row.role %></td>
+                <td class="align-middle"><%= row.uid %></td>
+                <td class="align-middle"><%= row.name %></td>
+                <td class="text-center align-middle"><%= row.role %></td>
                 <% } %>
-                <td class="text-center"><%= row.number %></td>
+                <td class="text-center align-middle"><%= row.number %></td>
                 <td class="text-center align-middle"><%- include('../partials/scorebar', {score: row.score_perc}); %></td>
-                <td class="text-center"><%= row.date_formatted %></td>
-                <td class="text-center" data-text="<%= row.duration_secs %>"><%= row.duration %></td>
-                <td class="text-center"><%= row.time_remaining %></td>
-                <td class="text-center">
+                <td class="text-center align-middle"><%= row.date_formatted %></td>
+                <td class="text-center align-middle" data-text="<%= row.duration_secs %>"><%= row.duration %></td>
+                <td class="text-center align-middle"><%= row.time_remaining %></td>
+                <td class="text-center align-middle">
                   <div class="dropdown">
                     <button type="button" class="btn btn-xs dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                       Action <span class="caret"></span>
                     </button>
+                    <div id="row<%= iRow %>PopoverSetTimeLimit"
+                         tabindex="0" data-toggle="popover" data-trigger="manual" data-container="body"
+                         data-html="true" data-placement="auto" title="Set Time Limit"
+                         data-content="
+                                       <p>
+                                         Time limit: <%= row.total_time %><br/>
```

Comment:
Maybe emphasize that this is the total time limit that the student has? I'm not sure how to best word this.
```suggestion
                                         Total time limit: <%= row.total_time %><br/>
```